ebill_paynet/components/api.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2019 Camptocamp SA
```

Comment:
why `components` if there's no component here?

Reply:
No components in the sense of https://github.com/OCA/connector/tree/13.0/component, true.